### PR TITLE
MUS-183 Write converter from noteevent to created interface

### PR DESCRIPTION
**Pull request by oliewa92**
Implement the converter which gets the data from the noteevents of the midi file to our data classes.

### CODE REVIEW

**Comment by paggerbua** [comment]
ok to test

File: catroidTest/src/org/catrobat/catroid/test/pocketmusic/note/trackgrid/TrackToTrackGridConverterTest.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+		Track track = TrackTestDataFactory.createSimpleTrack();
+		TrackGrid simpleTrackGrid = TrackGridTestDataFactory.createSimpleTrackGrid();
+
+		TrackGrid convertedTrackGrid = converter.convertTrackToTrackGrid(track, Project.DEFAULT_BEAT, Project
+				.DEFAULT_BEATS_PER_MINUTE);
+
+		assertTrue("Failed to convert simple Track.", convertedTrackGrid.equals(simpleTrackGrid));
+	}
+
+	public void testConvertTrackTihSeveralBreaks() {
```
Comment (joseffilzmaier) [inline catroidTest/src/org/catrobat/catroid/test/pocketmusic/note/trackgrid/TrackToTrackGridConverterTest.java]: I think this is a typo: testConvertTrack**Tih**SeveralBreaks

File: catroidTest/src/org/catrobat/catroid/test/pocketmusic/note/trackgrid/TrackToTrackGridConverterTest.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+import org.catrobat.catroid.pocketmusic.note.trackgrid.TrackGrid;
+import org.catrobat.catroid.pocketmusic.note.trackgrid.TrackToTrackGridConverter;
+import org.catrobat.catroid.test.pocketmusic.note.TrackTestDataFactory;
+
+public class TrackToTrackGridConverterTest extends AndroidTestCase {
+
+	@Override
+	protected void setUp() throws Exception {
+		super.setUp();
+	}
```
Comment (robertpainsi) [inline catroidTest/src/org/catrobat/catroid/test/pocketmusic/note/trackgrid/TrackToTrackGridConverterTest.java]: Remove me please!

File: catroid/src/org/catrobat/catroid/pocketmusic/note/trackgrid/TrackToTrackGridConverter.java [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class TrackToTrackGridConverter {
+
+	public TrackToTrackGridConverter() {
+	}
+
+	public TrackGrid convertTrackToTrackGrid(Track track, MusicalBeat beat, int beatsPerMinute) {
```
Comment (robertpainsi) [inline catroid/src/org/catrobat/catroid/pocketmusic/note/trackgrid/TrackToTrackGridConverter.java]: The `TrackToTrackGridConverter` is a (stateless) utility class. Make the [constructor private](https://github.com/Catrobat/Catroid/blob/develop/catroid/src/org/catrobat/catroid/utils/Utils.java#L88-L91) and the method static.

**Pull request merged**